### PR TITLE
Unknown persona coin

### PR DESCRIPTION
**Pull request by idijan**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Added support for UnknownPersona special case to Persona and PersonaCoin.
This is used in Outlook Web and Persona Card when an email sender cannot be authenticated.

Design:
image

### CODE REVIEW

File: common/changes/office-ui-fabric-react/unknownPersonaCoin_2018-05-08-13-17.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Added support for special casing UnknownPersona coin",
+      "type": "minor"
```
Comment (mtennoe) [inline common/changes/office-ui-fabric-react/unknownPersonaCoin_2018-05-08-13-17.json]: Its backwards compatible, so can be a patch

File: packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx [hunk 2]
```diff
@@ -195,11 +197,12 @@ export class PersonaCoinBase extends BaseComponent<IPersonaCoinProps, IPersonaSt
     const {
       allowPhoneInitials,
       primaryText,
+      showUnknownPersonaCoin
     } = props;
 
     const isRTL = getRTL();
 
-    imageInitials = imageInitials || getInitials(primaryText, isRTL, allowPhoneInitials);
+    imageInitials = (showUnknownPersonaCoin && '?') || imageInitials || getInitials(primaryText, isRTL, allowPhoneInitials);
```
Comment (mtennoe) [inline packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx]: lets move the ? to a const

File: common/changes/office-ui-fabric-react/unknownPersonaCoin_2018-05-08-13-17.json (hunk 1 above)
Comment (idijan) [reply common/changes/office-ui-fabric-react/unknownPersonaCoin_2018-05-08-13-17.json]: But it has a (non-breaking) API change, which qualifies it as minor according to https://github.com/OfficeDev/office-ui-fabric-react/blob/master/ghdocs/Contributing/ChangeFiles.md
>Any API change should qualify as at least as a minor change. When you add a new API in a backward compatible manner. i.e. Use of the new API is optional.

Comment (mtennoe) [reply common/changes/office-ui-fabric-react/unknownPersonaCoin_2018-05-08-13-17.json]: I stand corrected!

**Pull request merged**